src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -348,12 +348,12 @@ internal static void EnsureNullableAttributeExists(CSharpCompilation compilation
             }
         }
 
-        internal static bool ReportDefaultParameterErrors(
-            Binder binder,
+        internal static bool ReportDefaultParameterErrors(Binder binder,
```

Comment:
>(Binder binder, [](start = 57, length = 15)

nit: consider restoring original formatting (aligns with other members in this file)